roscala/src/main/scala/coop/rchain/rosette/prim/fixnum.scala

```diff
@@ -475,7 +480,7 @@ object fixnum {
     }
   }
 
-  private def checkFixnum(n: Int, elem: Seq[Ob]): Either[PrimError, Fixnum] =
```

Comment:
I don't understand the reason for removing the `private` modifier. If you're trying to get rid of a warning like "private method never called" or something along those lines then just delete the method entirely -- it's obviously needed in that case.

Reply:
Yes, you are right. 
Maybe it is a mistake of compiler.

Reply:
Done